activitysim/examples/prototype_arc/configs/joint_tour_scheduling.yaml

```diff
@@ -2,6 +2,7 @@ LOGIT_TYPE: MNL
 
 SPEC: tour_scheduling_joint.csv
 COEFFICIENTS: tour_scheduling_joint_coeffs.csv
+sharrow_skip: True
```

Comment:
These models have the `sharrow_skip` option set to true so they will not use sharrow.  This is not because sharrow is incompatible -- if it is enabled, it does work, and the results are correct.  But rather, it is because they specs for these models include a LOT of string comparisons, and string comparisons are slow in numba.  

It is possible to rewrite the spec to factor away all or nearly all string comparisons and achieve good performance, in a way that is 100% backward compatible. This has been done for the prototype_mtc model.  But rewriting specs to enhance performance for other example models is explicitly outside the scope of this PR.

I have added a note to each YAML file where `sharrow_skip` is used to explain why.  I also added to the documentation under `Using Sharrow > Sharrow Compatability and Limitations > Performance Considerations` (where the string comparison performance limitation is already documented) that explains that `sharrow_skip` can be used in these cases to disable sharrow until the spec has been updated for better performance.